recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
```

Comment:
I wonder if we should **only** run this on Appveyor? We could add something like `# [not win and environ['APPVEYOR'] != "True"]`. This would prevent someone from downloading a recipe and having it mess with their system.